packages/kit/src/core/node/index.js

```diff
@@ -1,44 +1,46 @@
 /**
  * @param {import('http').IncomingMessage} req
- * @returns {Promise<string | Uint8Array>}
+ * @returns {Promise<Uint8Array | string | null>}
  */
 export function getRawBody(req) {
 	return new Promise((fulfil, reject) => {
 		const h = req.headers;
 
 		if (!h['content-type']) {
-			fulfil(null);
-			return;
+			return fulfil(null);
 		}
 
 		req.on('error', reject);
 
 		const length = Number(h['content-length']);
 
-		/** @type {Uint8Array} */
-		let data;
-
-		if (!isNaN(length)) {
-			data = new Uint8Array(length);
+		// https://github.com/lukeed/polka/blob/next/packages/parse/index.js#L17
```

Comment:
do we need both of these links? I would just include the second one since it explains it whereas the first one doesn't really add any additional context